Doc/library/ssl.rst

```diff
@@ -869,7 +869,7 @@ Constants
 
    Whether the OpenSSL library has built-in support for *Next Protocol
    Negotiation* as described in the `NPN draft specification
-   <https://tools.ietf.org/html/draft-agl-tls-nextprotoneg>`_. When true,
+   <https://tools.ietf.org/html/draft-agl-tls-nextprotoneg-04>`_. When true,
```

Comment:
I dislike pointing old an old draft of 2012 :-(

https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation : "On July 11, 2014, ALPN was published as RFC 7301. ALPN replaces NPN"

Maybe we should point to https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation ?

@tiran, @alex: what do you think?

Reply:
Yeah, I think pointing to an article with the full context makes sense.

Reply:
Sure, I've updated it :)